front-end/bundle-configuration/default/runtime-config.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+export APPLICATION_SECRET=PPjOW0n2aV?s@6RdiNV@7/5xJhiiKTzk[VdHjkU9YHit8sLHoJ1rp0DCCn6b=lXt
```

Comment:
Let me create an issue for that. We'll add the comment once the video has been hosted.